ds/docker-entrypoint.sh

```diff
@@ -0,0 +1,297 @@
+#!/usr/bin/env bash
```

Comment:
So are you actually using this file or is this a sample?

Reply:
using it